packages/api-extractor/extends/tsdoc-base.json

```diff
@@ -71,6 +71,7 @@
 
 		"@betaDocumentation": true,
 		"@internalRemarks": true,
+		"@mixes": true,
```

Comment:
This can probably be removed now.

https://github.com/Qjuh/discord.js/blob/scripts/external-docs-main-branch/packages/builders/tsdoc.json

Can you maybe also add `@unstable` here, because of discord-api-types?

Reply:
`@unstable` should be defined in `tsdoc.json` for `discord-api-types` not here. The only reason `@mixes` got added here is because our `api-extractor` actually uses that tag for documentation decisions.
Alternatively the logic what to do with `@unstable` would have to be added to `api-extractor` too.

Reply:
And the tsdoc.json in builders is still needed for eslint to recognize the `@mixes` tag. But it should simply extend the `api-extractor` version.